packages/engine/src/config/worker_pool.rs

```diff
@@ -11,7 +11,7 @@ pub struct Config {
 impl Config {
     pub fn new(worker_base_config: WorkerConfig, max_num_workers: usize) -> Config {
         let _num_workers = std::cmp::min(num_cpus::get(), max_num_workers);
-        let num_workers = 1; // TODO: remove this
+        let num_workers = 1; // TODO: remove when multiple workers are fixed
```

Comment:
Do we need either of these here at all? Can we not just override it where we're setting it at the top-level of the binary?

I think we should set them in one place, so we should probably move the `std::cmp::min` up too

Reply:
It's just that the engine might run very slowly if the user accidentally sets a much higher number of workers than `num_cpus`